Templates/ScriptCanvasNode/Template/README.MD

```diff
@@ -5,7 +5,7 @@ Use this template to produce a Script Canvas project that may hold Script Canvas
 This guide will assume that you have created a gem called: `YourGem`
 
 ```
-scripts\o3de create-from-template -tp C:\dev\o3de\Templates\ScriptCanvasNode -dp C:\dev\o3de\Gems\YourGem\Code\Source\MyScriptCanvasNodes
+scripts\o3de create-from-template -tp C:\dev\o3de\Templates\ScriptCanvasNode -dp C:\dev\o3de\Gems\YourGem\Code\Source\MyScriptCanvasNodes -r gem_name YourGem
```

Comment:
Can you make the `README.md` extension lowercase, since it is lowercase in the template.json file?